packages/@office-iss/react-native-win32/package.json

```diff
@@ -51,7 +51,7 @@
     "stacktrace-parser": "^0.1.3",
     "use-subscription": "^1.0.0",
     "whatwg-fetch": "^3.0.0",
-    "ws": "^6.1.4"
+    "ws": "^7.4.6"
```

Comment:
Is this the way RN apps consume the native WS implementation, or is this a separate WS library?

Reply:
By looking at https://github.com/websockets/ws/tree/7.4.6, I can confirm this is a third-party JS-only library.

@dannyvv for my education, why is this library needed, if RNW already provides a native WS implementation?

Reply:
Most of the public developer tooling for RN is written in Javascript. Some of those will run a websocket server. I would look for usages of this in the "react-native" package.

It could be Android/iOS only, but we need to sync our packages with upstream dependencies because they can add or remove usages in their code.